.github/workflows/nightly.yml

```diff
@@ -1,7 +1,7 @@
 name: Nightly
 on:
   schedule:
-    - cron: '0 0 * * *' # Every day at midnight
+    - cron: '0 17 * * *' # Every day at 5pm GMT (9am PST / 10am PDT)
```

Comment:
I'm going to drop this back by an hour just so that the 30minute run happens before the typically scheduled meeting.

Reply:
Could you fix the commit message too?